### DISCUSSION TITLE
[ERLEDIGT] Konfiguration Homeassistant mit Homegear

### DISCUSSION THREAD

**Discussion by dcboy87** [SukramJ/aiohomematic#530] [Q&A]
Servus zusammen,
ich bin noch relativ neu in dem ganzen Thema und hab bei der Einrichtung noch Probleme. Homegear habe ich als Addon in HA installiert ([https://github.com/devRoemer/hassio-homegear-generic]) und da dann laut Wikieinträgen konfiguriert. In Homegear hab ich auch schon zwei Intertechno Geräte angelegt, kann also normal auf das Webinterface von Homegear zugreifen.

Das sind meine Einstellungen auf der ersten Seite der Integration:
![grafik](https://user-images.githubusercontent.com/45015463/178485730-c7dc1d12-6266-4589-965e-64f6b8a771c4.png)
die IP ist die meiner HA Installation

Das wäre die zweite Seite der Einrichtung:
![grafik](https://user-images.githubusercontent.com/45015463/178485996-e9e74b7e-ade7-49df-820a-a8a05e62df36.png)

Anschließend sagt er mir aber, dass er keine Verbindung herstellen kann. Hat hier jemand das gleiche Problem oder bin ich auf dem falschen Dampfer?!

LG Dominic

**Comment by SukramJ** [comment]
Ich kann nur raten, da ich Homegear nicht verwende.
Auf der Startseite des Homegear-Addons sollte der Hostname (*****-homegear) angezeigt werden. 
Verwende diesen anstatt der IP-Adresse

**Comment by dcboy87** [comment]
Danke dir für den Tipp, leider ändert das nichts. Evtl werd ich Homegear noch mal neu aufsetzten und die Config lassen wie sie ist.

**Reply by SukramJ** [reply]
Bitte mach dann ggf. hier ein Issue auf und fülle das Template aus.

**Reply by dcboy87** [reply]
ok, scheint kein Fehler von hahomematic zu sein, ich hab wohl wirklich zu forsch einige Ports beim Homegear Addon  geschlossen. In meinem Fall war es der Port 2001 für den RPC Server. Nun klappts und die Elemente tauchen auf
